disperser/dataapi/ejector.go

```diff
@@ -82,6 +84,11 @@ func (e *Ejector) Eject(ctx context.Context, nonsigningRate *OperatorsNonsigning
 
 	nonsigners := make([]*OperatorNonsigningPercentageMetrics, 0)
 	for _, metric := range nonsigningRate.Data {
+		// If nonsigningRateThreshold is set and valid, we will only eject operators with
+		// nonsigning rate >= nonsigningRateThreshold.
+		if e.nonsigningRateThreshold >= 10 && e.nonsigningRateThreshold <= 100 && metric.Percentage < float64(e.nonsigningRateThreshold) {
```

Comment:
what if e.nonsigningRateThreshold is 9, then regular SLA is enforced.
then if e.nonsigningRateThreshold is 50, then we enable the relaxed ejection.

Reply:
In first case: e.nonsigningRateThreshold will have no effect, it will just perform regular ejection

In second case: it'll eject an operator if 1) the operator violates SLA; **AND** 2) the operator's nonsigning rate >= 50%

Note: when it's regular ejection, only condition 1) is used